apps/wallet/src/ui/app/helpers/getSendOrSwapUrl.ts

```diff
@@ -0,0 +1,10 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+export function getSendOrSwapUrl(page: 'send' | 'swap', coinType: string, quoteAsset?: string) {
+	const encodedCoinType = encodeURIComponent(coinType);
+	const encodedQuoteAsset = quoteAsset ? encodeURIComponent(quoteAsset) : undefined;
+
+	const path = `/${page}?type=${encodedCoinType}`;
```

Comment:
I'd use URLSearchParams to build this which will be easier and less error-prone. You honestly won't even need this helper function at all with that